spec/controllers/routes_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe RoutesController, type: :controller do
+  let(:json_response) { JSON.parse(response.body, symbolize_names: true) }
+  let!(:organization) { create(:organization) }
+
+  describe 'POST #create' do
+    def create_route_call
+      post :create
+    end
+
+    context 'when user is authenticated' do
+      let!(:auth_user) { create_an_authenticated_user_with(organization, '1', 'android') }
+      let(:u_serializer) { UserSerializer }
+      before(:each) { create_route_call }
+
+      it 'does return success' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'does return a route id' do
+        expect(json_response[:id]).not_to be_nil
```

Comment:
You are not checking that the response is being serialize by the route serializer...